app/models/country.rb

```diff
@@ -25,6 +25,10 @@ def self.united_states
     where(name: "United States", iso: "US").first_or_create
   end
 
+  def self.netherlands
```

Comment:
There is also a place where this netherlands is referenced in `app/views/bikes/_stolen_checklist.html.haml`, and should be replaced with this.

Reply:
Updated! thanks - 939a2ea6